recipes/nmos-cpp/all/patches/avahi-mdnsresponder-target.patch

```diff
@@ -0,0 +1,93 @@
+diff --git a/Development/cmake/NmosCppDependencies.cmake b/Development/cmake/NmosCppDependencies.cmake
+index 58bf1e3c..c0a54295 100644
+--- a/Development/cmake/NmosCppDependencies.cmake
++++ b/Development/cmake/NmosCppDependencies.cmake
+@@ -257,6 +257,10 @@ if(${CMAKE_SYSTEM_NAME} STREQUAL "Linux")
+     # find Bonjour or Avahi compatibility library for the mDNS support library (mdns)
+     set(NMOS_CPP_USE_AVAHI ON CACHE BOOL "Use Avahi compatibility library rather than mDNSResponder")
+     if(NMOS_CPP_USE_AVAHI)
++        # third_party/cmake/FindAvahi.cmake uses the package name and target namespace 'Avahi'
```

Comment:
Is this from a PR or commit? might be worth taking the `.patch` version from github so it has the header and makes it a little move obvious why the "extra" bits are comming in

Reply:
It's from three commits so I've just added the `patch_source` instead in the _conandata.yml_.